specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -24,17 +24,25 @@ To see additional help and options, run:
 
 These are the global settings for the Machine Learning Services API.
 
-
 ``` yaml
 openapi-type: arm
-tag: package-2021-04-01
+tag: package-preview-2021-08
```

Comment:
even though I'm sure this was autogenerated to be in this format, can you update this to `package-2021-08-preview` to be consistent with our previous preview tags?

Reply:
Changed